compiler/hermes/include/hermes/core/Message.h

```diff
@@ -21,6 +21,7 @@
 #include <sstream>
 #include <string>
 #include <vector>
+#include "Severity.h"
```

Comment:
I think this is considered as a separate component ("hermes" vs "hermes-std" ), could you move this in separate PR?

Reply:
Please put this line above `#include <memory>` with an empty line separation.

Reply:
> I think this is considered as a separate component ("hermes" vs "hermes-std" ), could you move this in separate PR?

https://github.com/Samsung/ONE/pull/8002